qulice-maven-plugin/pom.xml

```diff
@@ -210,7 +210,7 @@
         <dependency>
             <groupId>org.sonatype.sisu</groupId>
             <artifactId>sisu-inject-plexus</artifactId>
-            <version>2.5.3</version>
+            <version>2.6.0</version>
```

Comment:
@mkordas 
it because when I changed source level to 1.7 I got this error http://stackoverflow.com/questions/10738609/what-does-supported-source-version-release-5-from-annotation-processor-me